rts/lra/lra-proxy/test/pom.xml

```diff
@@ -36,7 +36,9 @@
             <plugin>
                 <groupId>io.thorntail</groupId>
                 <artifactId>thorntail-maven-plugin</artifactId>
-                <version>${version.wildfly-thorntail}</version>
+<!--
+                <version>${version.thorntail}</version>
+-->
```

Comment:
same comment

Reply:
I'm not ready yet - I will clean up the PR when I've seen the TCK passing